documentation/asciidoc/computers/getting-started/network-installation.adoc

```diff
@@ -1,65 +1,28 @@
-
-image::images/beta-bootloader-1.png[width="80%"]
-
-Next, you need to select the boot order. All of the options are the same but will have a different boot order. Unless you have a reason to do otherwise, you should probably select “SD Card Boot.”
-
-image::images/beta-bootloader-2.png[width="80%"]
-
-Afterwards, select your media and follow xref:getting-started.adoc#installing-the-operating-system[the normal instructions] to burn the updated bootloader to your SD Card.
-
-When the SD Card has finished burning you should power off your Raspberry Pi and remove the existing SD Card and put it somewhere safe. Insert the card you just flashed with the Imager application into the Raspberry Pi and power it back on. The board LED will flash regularly, and the screen will go green to indicate that flashing the new beta bootloader was successful.
-
-[NOTE]
-====
-You can reset your device to the default bootloader by running the following command and rebooting,
-
-----
-$ sudo rpi-eeprom-update -f $(rpi-eeprom-update -l)
-----
-
-or by flashing the top “Bootloader” option instead of the “Beta Test Bootloader” option using the Raspberry Pi Imager.
-====
+IMPORTANT: Network installation is only enabled by default on Raspberry Pi 4 and Raspberry Pi 400.
 
 === Using Network Installation
 
-You will need a keyboard to make use of the network installation feature. While a Raspberry Pi 400 always has a keyboard "attached," if you're using a Raspberry Pi 4 you will need to plug in a USB keyboard.
+You will need a keyboard to make use of the network installation feature. While a Raspberry Pi 400 always has a keyboard "attached", if you're using a Raspberry Pi 4 you will need to plug in a USB keyboard.
 
 video::b1SYVpM9lto[youtube]
 
-You should also insert a **blank** SD Card into the SD card slot of your Raspberry Pi.
-
-NOTE: You should make sure there isn't a bootable SD Card, or USB drive, attached to your Raspberry Pi.
+Either insert a **blank** SD Card into the SD card slot of your Raspberry Pi, or press and hold the `SHIFT` key.
 
 Power on your Raspberry Pi. As always it will first look for an SD Card, and then a USB drive, to find bootable media. However if you have a keyboard attached, the Raspberry Pi will now show the network installation screen.
 
 image::images/network-install-1.png[width="80%"]
 
-In the background the Raspberry Pi is still looking for a bootable imagem, but you can now start a network installation by holding down the `SHIFT` key for 3 seconds. Confirm that you want to go ahead by pressing `SPACE`, and you will be prompted to connect your Raspberry Pi to the network with an Ethernet Cable. 
+In the background the Raspberry Pi is still looking for a bootable image, but you can now start a network installation by holding down the `SHIFT` key while powering on the device. Confirm that you want to go ahead by pressing `SPACE`, and you will be prompted to connect your Raspberry Pi to the network with an Ethernet Cable.
```

Comment:
Didn't one of the rpi-eeprom release notes say that the SPACE keypress was removed, or am I misremembering and/or getting confused?

Reply:
You're right - I think it's me that's confused.

Reply:
I've fiddled with this text